lib/use_case/base.rb

```diff
@@ -1,19 +1,19 @@
 # frozen_string_literal: true
 
-require 'dry-initializer'
-require 'types'
-require 'errors'
+require "dry-initializer"
+require "types"
+require "errors"
 
 module UseCase
   class Base
     extend Dry::Initializer
 
-    def self.call(*args, **kwargs)
-      new(*args, **kwargs).call
+    def self.call(...)
+      new(...).call
     end
 
     def call
       raise NotImplementedError
     end
```

Comment:
You can even do one of my favourite things in Ruby 3:

```suggestion
    def self.call(...) = new(...).call

    def call = raise NotImplementedError
```